src/parser.ts

```diff
@@ -669,13 +669,13 @@ export class Parser {
                     } else if (this.matchKeyword('class')) {
                         expr = this.parseClassExpression();
                     } else {
-                        this.throwUnexpectedToken(this.nextToken());
+                        throw this.unexpectedTokenError(this.nextToken());
```

Comment:
This is fine but if you were intent on using the same method as before for whatever reason, you could make throwUnexpectedToken return never and then return the result of that.